Mathlib/LinearAlgebra/LinearIndependent/Defs.lean

```diff
@@ -207,6 +207,9 @@ theorem LinearIndependent.comp (h : LinearIndependent R v) (f : ι' → ι) (hf
     LinearIndependent R (v ∘ f) := by
   simpa [comp_def] using Injective.comp h (Finsupp.mapDomain_injective hf)
 
+lemma LinearIndepOn.mono {t s : Set ι} (hs : LinearIndepOn R v s) (h : t ⊆ s) :
+    LinearIndepOn R v t := hs.comp _ <| Set.inclusion_injective h
+
```

Comment:
It makes me very sad that you are adding more **lemmas** to a file called `Defs.lean`.

Reply:
I agree... it's not your fault of course, but can you add a todo saying we should split the file?

Reply:
Done!